src/Components/ClusterHeader/ClusterHeader.spec.ct.js

```diff
@@ -43,6 +44,7 @@ describe('cluster page header', () => {
     // check last seen text
     cy.get(LAST_SEEN_FIELD).should('have.text', '24 Jul 2021 14:22 UTC');
   });
+  //FIXME spinner does not affect UUID/last seen?
   it('show spinner when in the loading state', () => {
```

Comment:
nit-pick: it is actually a skeleton, not spinner

Reply:
Thanks for pointing out. I should learn the right terms